travel.py

```diff
@@ -150,13 +250,68 @@ def flightPaths(fromloc, toloc):
         return response
 
 
-class TenantUserView(FlaskView):
+class TenantUserView(SwaggerView):
     """Class for storing user related information for a given tenant"""
 
     @api.route('/tenants/<tenant>/user/login', methods=['POST', 'OPTIONS'])
     @cross_origin(supports_credentials=True)
     def login(tenant):
-        """Login an existing user"""
+        """Login an existing user for a given tenant agent
+        ---
+        tags:
+        - tenants
+        parameters:
+            - name: tenant
+              in: path
+              required: true
+              schema:
+                type: string
+              example: tenant_agent_00
+              description: Tenant agent name
+        requestBody:
+            content:
+              application/json:
+                schema:
+                  type: object
+                  required:
+                   - user
+                   - password
+                  properties:
+                    user:
+                      type: string
+                      example: "user1"
+                    password:
+                      type: string
+                      example: "password1"
+        responses:
+            200:
+              description: Returns login data and query context information
+              content:
+                application/json:
+                  schema:
+                    type: object
+                    properties:
+                      context:
+                        type: array
+                        format: string
+                        example: ["A description of a KV operation"]
```

Comment:
Would be bit more work, but might be nice for the example Context outputs to give the most common (happy path) output?

Reply:
Yeah can do, especially when I extract this into a common object for reuse. We will have to pick a common example just to show something.

Obviously with the UI you can actually test the endpoint so you will see what the real response looks like anyways.